variables.tf

```diff
@@ -90,7 +72,7 @@ variable "lambdas" {
 
 variable "takeover" {
   description = "Create supported resource types to prevent malicious subdomain takeover"
-  default     = true
+  default     = false
```

Comment:
This is supposed to be `false` by default, i.e. in the dev environment, and `true` by default for production.

The reason for this is to prevent both a dev and a prod environment trying to take over the same S3 bucket / ElasticBeanstalk resulting in conflicts.

Reply:
Understood, thank you